### ISSUE TITLE
Location

### ISSUE THREAD

**Issue by manesh1x** [phamleduy04/texas-dps-scheduler#133]
Can you make the bot run even if a location isnt available within the miles set and keep looking for one, in hopes of the location coming back up. DPS removes the location that has no appointments available.

**Comment by phamleduy04** [comment]
Currently the app get the appointment when you run the app so there is no way to get the location if DPS removes it. I will try to find a workaround :)

**Comment by avinashj-aa** [comment]
I'm not able to see locations even if i give the exact dps location pincode. like plano, carrolton and tolar. it only detects 3 or 4 location even if the miles are set to 100

**Comment by phamleduy04** [comment]
@avinashj-aa have you try to use the official dps website to see if the location show up? If the official dps website didnt show up so does the app since they use the same system

**Comment by phamleduy04** [comment]
This can be achieved by `caching` all DPS location and use it locally.

**Comment by chappyboy** [comment]
Is there a way to get the preferred locations cached?

**Comment by phamleduy04** [comment]
> Is there a way to get the preferred locations cached?

The preferred location are cached

**Comment by chappyboy** [comment]
> > Is there a way to get the preferred locations cached?
> 
> The preferred location are cached

Yes, but I am looking to create a curated list of locations and the location which is close to me doesn't come in the options along with the other choices I want to keep.
